hivemind_core/protocol.py

```
@@ -286,6 +275,17 @@ def handle_client_disconnected(self, client: HiveMindClientConnection):
             self.callbacks.on_disconnect(client)
         except:
             LOG.exception("error on disconnect callback")
+
+        try:  # let the binary protocol know about it
+            self.binary_data_protocol.callbacks.on_disconnect(client)
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)